### ISSUE TITLE
Custom branding settings for hosted instances

### ISSUE THREAD

**Issue by ratsclub** [mataroablog/mataroa#75]
I started hosting my own mataroa instance and noticed that I had to replace a lot of templates to remove the mataroa branding for my own. Would it make sense to add values such as the following on `settings.py`?

- `INSTANCE_NAME` defaults to `Mataroa`
- `INSTANCE_DESCRIPTION` defaults to `Naked blogging platform, for minimalists.`

**Comment by sirodoht** [comment]
Yeah we should probably add things like these. We also have the email settings which need to be somehow configurable: https://github.com/mataroa-blog/mataroa/blob/1b7ca83f6e633dc1e54cbd9b0d9557731819b840/mataroa/settings.py#L190
